### ISSUE TITLE
Create tutorials on how to use `__check_auth` in interesting ways

### ISSUE THREAD

**Issue by briwylde08** [stellar/stellar-docs#627]
**Guide could include:**

- Supporting ed25519 and secp256r1 signers
- Deep dive into testing custom auth

**Comment by rahul-soshte** [comment]
@briwylde08 Assign me this issue.

**Comment by briwylde08** [comment]
You got it!

**Comment by od-hunter** [comment]
Can I be assigned this please? That’s if it’s open.

**Comment by briwylde08** [comment]
@rahul-soshte, are you still working on this issue?

@od-hunter, let's wait to work on this until we hear back from the original assignee.

**Comment by rahul-soshte** [comment]
@briwylde08 @od-hunter Yes actually I have completed this thing, gonna push the PR by tomorrow I think, just have to format it properly and then I am done with it.
https://github.com/rahul-soshte/stellar-docs/blob/fix-issue-627/docs/build/guides/conventions/check-auth-tutorials.mdx

**Comment by briwylde08** [comment]
Hi @rahul-soshte! Thank you so much for another contribution! @kalepail reviewed the PR this morning and had the following feedback: 
- The submission will be helpful to users, but does borrow a lot from https://github.com/stellar/soroban-examples/tree/main/account and https://github.com/kalepail/passkey-kit/blob/main/contracts/webauthn-wallet/src/lib.rs
- It would be great to see __check_auth used even more creatively 

That being said, your submission was accepted and we are happy to award you $200 worth of XLM for your contribution. 

Please confirm that the information we have on file for you from previous submissions is still correct so I can file the award!

Thank you and all the best!

**Comment by rahul-soshte** [comment]
Hi @briwylde08, thanks for the update. I am happy I have already contributed to 3 bounties and still counting till August 7th. Yes, the information you have on file that I provided in the previous submissions is correct. Just to clarify, was the award $200 or $300?

**Comment by briwylde08** [comment]
$200 worth of XLM for your submission [🙏](https://emojipedia.org/folded-hands)

**Comment by rahul-soshte** [comment]
Okay @briwylde08 thanks for the clarification.🤘
